docs/marginaleffects.ipynb

```diff
@@ -0,0 +1,371 @@
+{
```

Comment:
<ul><li>Add subsection ## Ratio estimates?</li><li>It would be great to give a short description of what the delta method is (just as as text box), specially for new users.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/py-econometrics/pyfixest/pull/473/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>